docs/book/component-gallery/experiment-trackers/mlflow.md

```diff
@@ -122,23 +122,22 @@ zenml stack register custom_stack -e mlflow_experiment_tracker ... --set
 ```
 {% endtab %}
 
-{% tab title="Secrets Manager (Recommended)" %}
+{% tab title="ZenML Secret (Recommended)" %}
```

Comment:
Secrets or Secret Stores
```suggestion
{% tab title="ZenML Secret Stores (Recommended)" %}
```

Reply:
I prefer to expose users to the "Secrets Store" notion as little as possible. The concept that they actually have to use is the "ZenML Secret", whereas the "secrets store" is something that should only come up during the initial deployment.